spec/sidekiq-instrument/client_middleware_spec.rb

```diff
@@ -40,30 +48,16 @@
     end
 
     context 'with WorkerMetrics.enabled true' do
-      let(:worker_metric_name) do
-        "sidekiq_instrument_trace_workers::in_queue"
-      end
       it 'increments the enqueue counter' do
-          Sidekiq::Instrument::WorkerMetrics.enabled = true
-          Redis.new.hdel worker_metric_name ,'my_other_worker'
-          MyOtherWorker.perform_async
-          expect(
-          Redis.new.hget worker_metric_name ,'my_other_worker'
-        ).to eq('1')
-      end
-    end
-
-    context 'with WorkerMetrics.enabled true and redis_config not provided' do
-      let(:worker_metric_name) do
-        "sidekiq_instrument_trace_workers::in_queue"
-      end
-      it 'increments the enqueue counter' do
-          Sidekiq::Instrument::WorkerMetrics.enabled = true
-          Redis.new.hdel worker_metric_name ,'my_other_worker'
-          MyOtherWorker.perform_async
-          expect(
-          Redis.new.hget worker_metric_name ,'my_other_worker'
-        ).to eq('1')
+        Sidekiq::Instrument::WorkerMetrics.enabled = true
+        MyOtherWorker.perform_async
+        expect(
+        Redis.new.hget(worker_metric_name ,'my_other_worker')
+      ).to eq('1')
+        MyOtherWorker.perform_async
+        expect(
+        Redis.new.hget(worker_metric_name ,'my_other_worker')
+      ).to eq('2')
```

Comment:
Can we clean up the weird line-breaking and comma spacing from those old tests to match our `expect()` calls [elsewhere](https://github.com/enova/sidekiq-instrument/pull/24/files?diff=split&w=0#diff-bf3ae6f4843f969152133a71a1b11eeda8c87da740d816f88e1a9f4220fae5d4R64)?
```suggestion
        expect(Redis.new.hget(worker_metric_name, 'my_other_worker')).to eq('1')
        MyOtherWorker.perform_async
        expect(Redis.new.hget(worker_metric_name, 'my_other_worker')).to eq('2')
```